### DISCUSSION TITLE
Search icon should be moved over to the left pane

### DISCUSSION THREAD

**Discussion by madbrain76** [home-assistant/frontend#12020] [Other feature requests]
This is a followup to this issue : https://github.com/home-assistant/frontend/discussions/12018

The magnifying glass icon is only shown in the configuration screen, on the top right side. When one clicks this icon, and the search bar comes up, however, it is capable of searching the entire UI and state of HomeAssistant, not just the configuration.

IMO, it would make much more sense if the icon was visible at all times in the UI, for example in the left pane, than just in the configuration screen.

**Comment by matthiasdebaat** [comment]
We're improving the Quick Bar by making it easier to find and turning it into a [single search bar](https://github.com/home-assistant/frontend/discussions/11511). In one of our concepts, we have the search option in the side menu. This isn't final yet, but I agree it's the most logical place.
